### PR TITLE
Adding meng's rubbish.

### PR DESCRIPTION
**Pull request by ymloh**
Good rubbish is good.

### CODE REVIEW

**Comment by notevenagoat** [comment]
Wow, much content.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # buddytime
 this is where we party 24/7
+## This is also fine.
```
Comment (notevenagoat) [inline README.md:3]: such addition

**Review by notevenagoat — COMMENTED** [review]
i am reviewing a merged changed comment ?

File: README.md (hunk 1 above)
Comment (ymloh) [reply README.md:3]: much complex

**Pull request merged**